api/users/models.py

```diff
@@ -86,6 +89,10 @@ class FFAdminUser(AbstractUser):
     last_name = models.CharField(_("last name"), max_length=150)
     google_user_id = models.CharField(max_length=50, null=True, blank=True)
     github_user_id = models.CharField(max_length=50, null=True, blank=True)
+    is_subscribed = models.BooleanField(
```

Comment:
This field name is a bit misleading, can we be more explicit? Something like `subscribed_to_mailing_list` or `subscribed_to_mailer_lite` ?